.env.example

```diff
@@ -1,4 +1,4 @@
-OPENAI_API_KEY=
-QDRANT_URL=         #Defaults to http://localhost:6334
-RUST_LOG=           #Logging levels: "error", "warn", "info", "debug", "trace"
-WEBSERVER_PORT=     #Defaults to 3000
\ No newline at end of file
+OPENAI_API_KEY=     #REQUIRED
+WEBSERVER_PORT=     #REQUIRED
+CHROMA_URL=         #Defaults to http://localhost:8000
```

Comment:
Where does this get used to set the Chroma client? I'm not seeing any other references to this in the code. Is it being consumed by the Chroma rust client?

Reply:
The Chromadb-rs lib uses `CHROMA_URL` env as fallback by default.
https://github.com/open-sauced/repo-query/blob/37bd46815bc9eb765bb9a271783023c1f1ad9c5c/src/db/chroma.rs#L21